x-pack/plugins/cases/public/components/app/use_available_owners.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApplicationStart } from 'kibana/public';
+import { useKibana } from '../../common/lib/kibana';
+
+/**
+ *
+ * @param level : 'crud' | 'read' (default: 'crud')
+ *
+ * `securitySolution` owner uses cases capability feature id: 'securitySolutionCases'; //owner
+ * `observability` owner uses cases capability feature id: 'observabilityCases';
+ * both solutions use `crud_cases` and `read_cases` capability names
+ **/
+
+export const useAvailableCasesOwners = (level: 'crud' | 'read' = 'crud'): string[] => {
```

Comment:
Just a reminder but let's write some tests for this function.

Reply:
Sorted :D